common/config/config.json.template

```diff
@@ -11,16 +11,16 @@
   "liveDvrWindow" : 100,
   "logLevel" : "debug",
   "logToConsole" : true,
-  "logFileName" : "/var/log/dvr_worker.log",
+  "logFileName" : "/var/log/kDvrWorker.log",
```

Comment:
I wasn't talking about SaaS path, the standard I mentioned is our on-prem and community standard.